wgpu-core/src/device/mod.rs

```diff
@@ -1657,6 +1657,15 @@ impl<A: HalApi> Device<A> {
                         view_dimension: view.desc.dimension,
                     });
                 }
+
+                let mip_level_count = view.selector.levels.end - view.selector.levels.start;
```

Comment:
If this is a range, this could be `view.selector.levels.len()`.

Reply:
It is a range, but `len` is from `Iterator` and it returns an `usize` instead of `u32`. I'm not sure if we need that type conversion.

Reply:
Man why does Range have to be so weird. Honestly it's a wash, lets just merge this.